prep-html-syntax/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <title>HTML Syntax</title>
```

Comment:
Hi Alvaro! Be sure to put your `<title></title>` inside of your `<head></head>` element

Reply:
https://github.com/alvarodiazjr/prep-0422-code-solutions/pull/2/commits/3686229c9fbfcbbe0359da1fce324d388ef43022